### DISCUSSION TITLE
Load Pytorch Model Weights in Python Backend

### DISCUSSION THREAD

**Discussion by maxi-w** [triton-inference-server/server#6108] [Q&A] [answered]
Hey, I have a model deployed as python backend and was wondering if and how I can suply additional files like a model.pth with that python backend?

One way to solve this would be to deploy the model as e.g. ONNX but that comes with some extra work of seperating out the pre-/post-processing and combining the steps.

**Comment by hasnainkothawala** [comment, marked as answer]
Not sure if I understand you correctly but this how i did it , 

I wrote this in my model.py 

```
def initialize(self, args):
	try:
		self.image_model_path = Path("/mnt/models/detectron-service/1/resources/v1.0/model_final.pth")
		self.image_confidence_threshold = 0.8
		self.detectron_model_weights_file = ("COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml")
		self.detectron2_config = None
```


And added the model.pth file in the same location :
